recipe/meta.yaml

```
@@ -1,26 +1,26 @@
 {% set name = "sphinx-basic-ng" %}
-{% set version = "1.0.0b1" %}
+{% set version = "1.0.0.0beta2" %}
```

```suggestion
{% set version = "1.0.0b2" %}
```
The conda-forge recipe is fine with it, see https://github.com/conda-forge/sphinx-basic-ng-feedstock/blob/main/recipe/meta.yaml#L2-L9

---

We should follow PyPI https://pypi.org/project/sphinx-basic-ng/#history

---

Technically, it's a beta version. I wonder why we build it, because we don't support alpha, beta, rc1, rcN, etc. versions.

---

it's a dependency for furo package

---

https://github.com/pradyunsg/furo/blob/e2cbfce534e434c9743fae85839790ffb8fea2a2/pyproject.toml#L19

---

It's a question then if we should build `furo` because it has an unstable dependency